tests/integration/test_search_views.py

```diff
@@ -59,3 +62,15 @@ def test_search_jobs_is_there(app):
 def test_search_falls_back_to_hep(app):
     with app.test_client() as client:
         assert client.get('/search').status_code == 200
+
+
+@patch('inspirehep.modules.search.query.current_app')
+def test_search_logs(current_app_mock, app):
+    def _debug(log_output):
+        assert all(
+            [element in log_output for element in ("filter", "fields", "size")]
```

Comment:
I think this is always going to pass, no matter what fields you put here. What went wrong with the previous approach?